app/javascript/channels/room_channel.js

```diff
@@ -0,0 +1,40 @@
+import consumer from './consumer'
+
+function formatDate(date) {
+    return date.getFullYear() + "-" + ("0" + (date.getMonth() + 1)).slice(-2) + "-" + ("0" + date.getDate()).slice(-2) +
+        " " + ("0" + date.getHours()).slice(-2) + ":" + ("0" + date.getMinutes()).slice(-2);
+
+}
+
+$(function () {
+    $('[data-channel-subscribe="room"]').each(function (index, element) {
```

Comment:
In some cases, this seems seems to stop working after actions that reload the page without killing the WebSocket connection, like the `redirect_to` when trying to send an empty message, navigating to `My Contacts` and back in the navbar or navigating at the bottom in the mobile variant.
If this happens, one does not receive new messages, but sending still works, because it uses a `POST` request. Also, if this happens, the message container is not scrolled to the bottom.
Only if reloading using `Crtl-r` or similar, that also kills or reloads the ws connection, it works again.

Reply:
Fixed